.env.example

```diff
@@ -21,8 +21,9 @@
 # -- Output full compiler stats
 # WEBPACK_STATS=errors-only
 #
-# -- Enable server-side redis caching when logged out
-# ENABLE_SERVER_SIDE_CACHE=false
+# -- Enable redis caching when logged out. Must be connected to VPN.
```

Comment:
Is this accurate? (I think this is for local development so I'd imagine we should actually have this on by default, going forward).

Reply:
Would rather not increase friction (due to VPN requirements). Also keeping the uncached experience is better for dev! Easy enough to toggle on.

Reply:
You don't need a VPN (to access Force's staging Redis instance) to have this caching enabled locally - we run Redis on our local machines, no?

Reply:
Ah you know, you're right. I just remembered the volt bridge; never needed to be on the VPN to use it. I'll update the comment!